### DISCUSSION TITLE
How to use enum with string

### DISCUSSION THREAD

**Discussion by vishwas-jadav** [prisma/prisma#8244] [Q&A] [answered]
I want to use 'Require Major Work','Require Parts/Equipments','Require Service Center' values in enum for mysql but I am getting :  **Error validating: This line is not an enum value definition.** , Can anyone help ?

**Comment by ryands17** [comment, marked as answer]
@vishwas-jadav 👋 

Spaces are not allowed when using enums at the moment. You can write it as follows:

```prisma
enum Type {
  REQUIRE_MAJOR_WORK
  REQUIRE_PARTS_EQUIPMENTS
  REQUIRE_SERVICE_CENTER
}
```

**Reply by danielweil** [reply]
ANy way to use special caracters and translate what is in the database to the client?

**Reply by harensarma** [reply]
you may edit the raw migration file after npx prisma migrate dev --create-only

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
